bin/deploy

```diff
@@ -7,16 +7,16 @@ SCRIPT_DIR=$(cd "$(dirname "$0")"; pwd)
 # shellcheck disable=SC1090
 source "${SCRIPT_DIR}/../etc/settings.cfg.sh"
 
-EUPS_VERSION=${EUPS_VERSION:-2.1.4}         # Version of EUPS to install
-EUPS_GITREV=${EUPS_GITREV:-""}
-EUPS_GITREPO=${EUPS_GITREPO:-https://github.com/RobertLuptonTheGood/eups.git}
+LSST_EUPS_VERSION=${LSST_EUPS_VERSION:-2.1.4}
+LSST_EUPS_GITREV=${LSST_EUPS_GITREV:-""}
+LSST_EUPS_GITREPO=${LSST_EUPS_GITREPO:-https://github.com/RobertLuptonTheGood/eups.git}
 # force Python 3
 LSST_PYTHON_VERSION=3
-MINICONDA_VERSION=${MINICONDA_VERSION:-4.3.21}
-MINICONDA_BASE_URL=${MINICONDA_BASE_URL:-https://repo.continuum.io/miniconda}
-CONDA_CHANNELS=${CONDA_CHANNELS:-""}
-GIT_VERSION=${GIT_VERSION:-2.16.1}          # Version of git to install
-LFS_VERSION=${LFS_VERSION:-2.3.4}           # Version of git-lfs to install
+LSST_MINICONDA_VERSION=${LSST_MINICONDA_VERSION:-4.5.4}
+LSST_MINICONDA_BASE_URL=${LSST_MINICONDA_BASE_URL:-https://repo.continuum.io/miniconda}
+LSST_CONDA_CHANNELS=${LSST_CONDA_CHANNELS:-""}
+LSST_GIT_VERSION=${LSST_GIT_VERSION:-2.16.1}
+LSST_LFS_VERSION=${LSST_LFS_VERSION:-2.3.4}
```

Comment:
Is it worth bumping lfs at the same time? I'm using 2.4.2.

Reply:
Sure. Might as well bump git too even though I'd like to eventually remove it from lsstsw.